templates/aem-tools/deploy-artifact.sh.epp

```diff
@@ -7,6 +7,8 @@ if [ "$#" -ne 7 ]; then
   exit 1
 fi
 
+PATH=/opt/puppetlabs/bin:/opt/puppetlabs/puppet/bin:$PATH
```

Comment:
This should be moved to aem-aws-stack-provisioner since the curator doesn't need to know about path, it should only know that the command is available at this layer.